packages/desktop/lib/routers/actions/resetRouterForAppContext.ts

```diff
@@ -0,0 +1,16 @@
+import { AppContext } from '@core/app/enums'
+
+import { getRouterForAppContext, getSubroutersForAppContext } from '../utils'
+
+export function resetRouterForAppContext(context: AppContext, resetSubrouters = false): void {
+    const router = getRouterForAppContext(context)
+    if (router) {
+        if (resetSubrouters) {
+            getSubroutersForAppContext(context).forEach((subrouter) => subrouter.reset())
+        }
```

Comment:
Nothing to do here, just a small note:

This could be made recursive if we have sub-sub-routers (not needed for desktop, but for mobile this might be handy).